spec/spec_helper.rb

```diff
@@ -130,6 +130,15 @@ def restart_driver
     ActionController::Base.perform_caching = caching
   end
 
+  # Show javascript errors in test output with `js_debug: true`
+  config.after(:each, :js_debug) do
+    errors = page.driver.browser.manage.logs.get(:browser)
+    if errors.present?
+      message = errors.map(&:message).join("\n")
+      puts message
+    end
+  end
+
```

Comment:
nice!!!!

Reply:
Yeah this is really nice for debugging JS locally in feature specs :smile: